### ISSUE TITLE
[Flaky Build] VisualStudioSetup can build before the Microsoft.Build.Tasks.CodeAnalysis task, causing failures

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#6344]
VisualStudioSetup project builds before the Microsoft.Build.Tasks.CodeAnalysis task, causing the following build error on clean machines:

``` 
"C:\Users\davidjenkins\Documents\GitHubVisualStudio\roslyn\Roslyn.sln" (default
 target) (1) ->
"C:\Users\davidjenkins\Documents\GitHubVisualStudio\roslyn\src\VisualStudio\Set
up\VisualStudioSetup.csproj" (default target) (86) ->
(GeneratePkgDef target) ->
  C:\Program Files (x86)\MSBuild\Microsoft\VisualStudio\v14.0\VSSDK\Microsoft.V
sSDK.targets(911,5): error : CreatePkgDef : error : ProvideRoslynBindingRedirec
tionAttribute: Could not load specified assembly: 'Microsoft.Build.Tasks.CodeAn
alysis' reason: Could not load file or assembly 'file:///C:\Users\davidjenkins\
Documents\GitHubVisualStudio\roslyn\Binaries\Debug\Microsoft.Build.Tasks.CodeAn
alysis.dll' or one of its dependencies. The system cannot find the file specifi
ed. [C:\Users\davidjenkins\Documents\GitHubVisualStudio\roslyn\src\VisualStudio
\Setup\VisualStudioSetup.csproj]
```

**Comment by davkean** [comment]
Didn't @tannergooding fix this?

**Comment by jasonmalinowski** [comment]
This was fixed by #6515, and various detritus cleaned up in #6661.
